fuel/src/main/kotlin/com/github/kittinunf/fuel/core/interceptors/RedirectionInterceptor.kt

```diff
@@ -1,15 +1,13 @@
 package com.github.kittinunf.fuel.core.interceptors
 
-import com.github.kittinunf.fuel.Fuel
-import com.github.kittinunf.fuel.core.Encoding
-import com.github.kittinunf.fuel.core.FuelError
-import com.github.kittinunf.fuel.core.Request
-import com.github.kittinunf.fuel.core.Response
+import com.github.kittinunf.fuel.core.*
+import java.net.MalformedURLException
+import java.net.URL
 import javax.net.ssl.HttpsURLConnection
 
 class RedirectException() : Exception("Redirection fail, not found URL to redirect to")
 
-fun redirectResponseInterceptor() =
+fun redirectResponseInterceptor(manager: FuelManager) =
```

Comment:
The reason I pass the whole manager is that the subsequent requests will then be using the same baseHeaders, keystore, socketFactory, hostnameVerifier, executor etc. It seems to me that this is what one would expect. Otherwise any redirected requests will all be handled by FuelManager.instance.